server/storage/wal/wal_test.go

```diff
@@ -195,9 +196,8 @@ func TestOpenAtIndex(t *testing.T) {
 	w.Close()
 
 	emptydir := t.TempDir()
-	if _, err = Open(zaptest.NewLogger(t), emptydir, walpb.Snapshot{}); err != ErrFileNotFound {
-		t.Errorf("err = %v, want %v", err, ErrFileNotFound)
-	}
+	_, err = Open(zaptest.NewLogger(t), emptydir, walpb.Snapshot{})
+	require.ErrorContains(t, err, ErrFileNotFound.Error())
```

Comment:
This is not a proper way to test error. Please use `error.Is`.

Reply:
thx & updated.